src/main/java/org/testng/TestRunner.java

```diff
@@ -658,6 +659,11 @@ public long getTimeOut() {
         return 0;
       }
 
+      @Override
+      public void setLatch(CountDownLatch latch) {
+
```

Comment:
Sure Cedric. I will do that. I don't think this PR is good enough to be merged. I was kind of looking for feedback around how to solve the problem that the issue was displaying. Since this involves the Graph part of TestNG I am trying to get more understanding. I thought I would first show my changes and check with you and @juherr on whether I am going in the right direction.